src/FormLayout/_Abstract.php

```diff
@@ -40,27 +35,27 @@ public function addField($name, $decorator = null, $field = null)
             $field = ['type' => $field];
         }
 
-        if ($name) {
-            $existingField = $this->form->model->hasField($name);
-        }
-
         try {
-            if (!$existingField) {
+            if (!$this->form->model->hasField($name)) {
                 // Add missing field
                 if ($field) {
                     $field = $this->form->model->addField($name, $field);
```

Comment:
If you are anyway modifying maybe use one-liner here?

```
$field = $this->form->model->addField($name, $field ?: []);
```

Reply:
This was not intended to be fixed by this PR... But I will fix this now.

Reply:
Done - tests are passing even without `?: []` - this needs to be decised later if seed should support `null` or not - I already created an issue for it - https://github.com/atk4/core/issues/226 - but passing null/string seems to be used a lot - todo for later.

Ok otherwise, can you approve?